arangod/Cluster/FollowerInfo.cpp

```diff
@@ -542,3 +543,30 @@ VPackBuilder FollowerInfo::newShardEntry(VPackSlice oldValue) const {
   }
   return newValue;
 }
+
+uint64_t FollowerInfo::newFollowingTermId(ServerID const& s) noexcept {
+  uint64_t i = 0;
+  // We want the random number to be non-zero:
+  do {
+    i = RandomGenerator::interval(UINT64_MAX);
+  } while (i == 0);
+  try {
+    _followingTermId[s] = i;
+  } catch(std::bad_alloc const& exc) {
+    i = 1;   // I assume here that I do not get bad_alloc if the key is
+             // already in the map, since it then only has to overwrite
+             // an integer, if the key is not in the map, we default to 0.
```

Comment:
The comment here says that we default to `0`, but we seem to default to `1` in reality.